### PR TITLE
[Clock] Implement PSR-20

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Anticipating the creation of psr/clock v1.0
https://github.com/php-fig/fig-standards/pull/1289

/cc @heiglandreas

### CODE REVIEW

**Comment by 94noni** [comment]
You loose no time 🕐 hahaha

File: src/Symfony/Component/Clock/ClockInterface.php [hunk 1]
```diff
@@ -11,10 +11,12 @@
...
 
+use Psr\Clock\ClockInterface as PsrClockInterface;
+
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-interface ClockInterface
+interface ClockInterface extends PsrClockInterface
 {
     public function now(): \DateTimeImmutable;
```
Comment (stof) [inline src/Symfony/Component/Clock/ClockInterface.php]: this method should be removed as it overrides the parent one removing the phpdoc now.

Comment (nicolas-grekas) [reply src/Symfony/Component/Clock/ClockInterface.php]: updated!

**Comment by derrabus** [comment]
Shouldn't we require `psr/clock` in the root composer.json as well?

**Comment by derrabus** [comment]
Is this for 6.2 or 6.3? The PR description says 6.2, but you're targeting the 6.3 branch.

**Comment by nicolas-grekas** [comment]
Right, target + composer.json updated.

**Comment by Jean85** [comment]
1.0 is tagged! :tada: just restart the CI :smile:

**Review by chalasr — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Rebased, ready !

**Review by xabbuh — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**